### PR TITLE
【Hackathon 5th No.45】convert torch.cuda.comm.broadcast, torch.cuda.device -part

### PR DESCRIPTION
**Pull request by longranger2**
### PR Docs
https://github.com/PaddlePaddle/docs/pull/6217

### PR APIs

- [x]  convert torch.cuda.comm.broadcast
- [x]   convert torch.cuda.device

### CODE REVIEW

**Comment by longranger2** [comment]
@zhwesky2010 没有找到这个耶：torch.nn.modules.batchnorm._NormBase
image

File: tests/test_cuda_comm_broadcast.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# limitations under the License.
+
+import textwrap
+
+from apibase import APIBase
+
+obj = APIBase("torch.cuda.comm.broadcast")
+
+
+def test_case_1():
```
Comment (zhwesky2010) [inline tests/test_cuda_comm_broadcast.py:22]: @longranger2 你可能需要本地测试一下，因为目前CI上是CPU的torch

**Comment by zhwesky2010** [comment]
> @zhwesky2010 没有找到这个耶：torch.nn.modules.batchnorm._NormBase
> 
> image

以实际代码运行为判断标准，不要以文档为准

File: tests/test_cuda_comm_broadcast.py (hunk 1 above)
Comment (longranger2) [reply tests/test_cuda_comm_broadcast.py:22]: 很奇怪，本地GPU环境也是报错
image

**Comment by luotao1** [comment]
@longranger2 单侧还没有过
